Lib/uuid.py

```diff
@@ -949,6 +949,8 @@ def main():
     parser.add_argument("-N", "--name",
                         help="The name used as part of generating the uuid. "
                         "Only required for uuid3/uuid5 functions.")
+    parser.add_argument("-C", "--count", metavar="NUM", type=int, default=1,
```

Comment:
Should we really have a default value? Otherwise, `python -m uuid -C` would be allowed

Reply:
Oh, I guess I forgot about argparse :')